storage/integration_test.go

```diff
@@ -744,6 +746,127 @@ func TestIntegration_ObjectsRangeReader(t *testing.T) {
 	}
 }
 
+func TestIntegration_ObjectReadGRPC(t *testing.T) {
+	// TODO: Remove this once v2 rollout is complete.
+	t.SkipNow()
+
+	ctx := context.Background()
+	client, err := newClientWithGRPC(ctx)
+	if err != nil {
+		t.Error(err)
+	}
+	defer client.Close()
+	bkt := client.Bucket(grpcBucket)
+
+	objName := uidSpace.New()
+	obj := bkt.Object(objName)
+	contents := []byte("Hello, world this is a grpc request")
+
+	if err := retry(ctx, func() error {
+		w := obj.NewWriter(ctx)
```

Comment:
Should we make a separate client to write with so that this does not silently use gRPC once we have writer implemented? Or I guess it would be fine to do both in one test as long as it's clear.

Reply:
Great idea. I figured I'd punt on that to when we implemented Write w/gRPC, but might as well start off correct.

Reply:
Removed and refactored into a separate function that creates a new HTTP client. Done!